common/tests/org.jboss.tools.common.oauth.core.test/pom.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.jboss.tools.common</groupId>
+		<artifactId>tests</artifactId>
+		<version>3.16.0-SNAPSHOT</version>
+	</parent>
+	<groupId>org.jboss.tools.common.tests</groupId>
+	<artifactId>org.jboss.tools.common.oauth.core.test</artifactId>
+	<packaging>eclipse-test-plugin</packaging>
+
+   <build>
```

Comment:
remove entierly as JUnit5 single unit test should be ok now.

Reply:
went too fast. the build fails because no unit test found. please revert and remove only the skip configuration.